policyengine_canada/variables/gov/provinces/mb/tax/income/credits/tuition_amount/mb_tuition_amount_credit.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_canada.model_api import *
+
+
+class mb_tuition_amount_credit(Variable):
+    value_type = float
+    entity = Person
+    label = "Manitoba tuition amount credit"
+    definition_period = YEAR
+    defined_for = "mb_tuition_amount_credit_eligible"
+
+    def formula(person, period, parameters):
+        p = parameters(
+            period
+        ).gov.provinces.mb.tax.income.credits.tuition_amount
+
+        tuition = person("tuition_expenses", period)
```

Comment:
the legal code says it's divided by the lowest federal marginal rate

Reply:
The legal code differes from the form.